src/components/LoginForm/LoginForm.jsx

```diff
@@ -98,7 +106,7 @@ export const LoginForm = () => {
                 id="password"
                 placeholder={i18next.t('login.form.password')}
                 type={show ? 'text' : 'password'}
-                {...register('password', { required: true })}
+                {...register('password', { required: 'Password is required' })}
               />
```

Comment:
you have to add additional rules for passwords:
<img width="312" alt="image" src="https://user-images.githubusercontent.com/39521158/180749553-151ea097-5e41-4c4b-91b3-044de0dbd8a0.png">

Reply:
I asked. Only validation is "required".